Floofbot/Modules/Tag.cs

```diff
@@ -4,215 +4,191 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 
-namespace Floofbot
+namespace Floofbot.Modules
 {
-    public partial class Utility
+    [Group("tag")]
+    [RequireContext(ContextType.Guild)]
+    [RequireUserPermission(Discord.GuildPermission.AttachFiles)]
+    public class Tag : ModuleBase<SocketCommandContext>
     {
-        [Group("tag")]
-        [RequireContext(ContextType.Guild)]
-        [RequireUserPermission(Discord.GuildPermission.AttachFiles)]
-        public class Tag : ModuleBase<SocketCommandContext>
-        {
-            SqliteConnection dbConnection;
-            List<string> pages;
-
-
-            public Tag()
-            {
-                dbConnection = new SqliteConnection(new SqliteConnectionStringBuilder {
-                    DataSource = "botdata.db"
-                }.ToString());
-                dbConnection.Open();
-            }
+        private SqliteConnection dbConnection;
+        private static readonly Discord.Color embedColor = Color.Magenta;
 
-            [Command("add")]
-            [Priority(0)]
-            [RequireUserPermission(GuildPermission.AttachFiles)]
-            public async Task Add(string Tag, [Remainder] string Content = null)
-            {
-                try {
-                    if (Content != null) {
-                        Regex rgx = new Regex("[^a-zA-Z0-9 -]");
-                        Tag = rgx.Replace(Tag, "").ToLower();
-                        string sql = @"INSERT into Tags (TagID,UserID, Content)
-                               VALUES ($TagId, $UserID, $Content)";
-
-                        SqliteCommand command = new SqliteCommand(sql, dbConnection);
-                        command.Parameters.Add(new SqliteParameter("$TagId", Tag.ToString() + ":" + Context.Guild.Id.ToString()));
-                        command.Parameters.Add(new SqliteParameter("$UserID", Context.User.Id.ToString()));
-                        command.Parameters.Add(new SqliteParameter("$Content", Content));
-
-                        var result = command.ExecuteScalar();
+        public Tag()
+        {
+            dbConnection = new SqliteConnection(new SqliteConnectionStringBuilder {
+                DataSource = "botdata.db"
+            }.ToString());
+            dbConnection.Open();
+        }
 
-                        EmbedBuilder builder = new EmbedBuilder() {
-                            Description = $"💾 Added Tag `{Tag}`",
-                            Color = Color.Magenta
-                        };
-                        await Context.Channel.SendMessageAsync("", false, builder.Build());
+        [Command("add")]
+        [Priority(0)]
+        [RequireUserPermission(GuildPermission.AttachFiles)]
+        public async Task Add(string Tag, [Remainder] string Content = null)
```

Comment:
So I know this is kind of an artifact from the previous iteration of this code, but the capitalized variable names are throwing me off a bit.

Reply:
fixed